src/extension/background.js

```diff
@@ -255,6 +255,30 @@ async function updateHelpContent() {
   }
 }
 
+/**
+ * Checks if the view source popp needs to be openeded, and opens it if necessary.
+ * @param {*} id The tab ID
+ */
+function checkViewSource(id) {
+  chrome.tabs.get(id, (tab = {}) => {
+    if (!tab.url) return;
+    try {
+      const u = new URL(tab.url);
+      const vds = u.searchParams.get('view-doc-source');
```

Comment:
Why is the parameter not just called `view-source`?

Reply:
To not conflict with the "real" view source. It is not the source of the page but the source document.

Reply:
Should we also rename the folder to `view-doc-source` then?

Reply:
ok, I see the confusion. I thought the folder name does not matter and we would know the difference. But I am going to rename it.